### ISSUE TITLE
Visualize CSS breakpoints in patternplate demo interface

### ISSUE THREAD

**Issue by marionebl** [patternplate/patternplate#101]
Media queries and responsive design remains a big source of misunderstanding between client, creatives and development teams. In order to help with communication about this topic, patternplate should provide means to visualize and document breakpoints for pattern demos. 

**Goals**
- [ ] Provide means to extract information about applicable breakpoints from CSS used for the current pattern demo
- [ ] Show applicable breakpoints (a.k.a: upper and lower media query bounds) as guides on the resize rulers already available in patternplate client. Both `*-width` and `*-height` breakpoints should be analyzed
- [ ] Clicking on breakpoint guides should resize the demo to the appropriate dimensions

**Non-Goals**
- Writing access to CSS via the patternplate UI is no goal of this feature

**Comment by marionebl** [comment]
Blocked by rulers/resizing currently being not implemented, closing for now.
